helloworld/src/components/Card.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+
+export default class Card extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: "Ly",
+      lastName: "Doan Thi"
+    };
+  }
+
+  sendToParent = () => {
+    this.props.onReceiveData(this.props); //Call thang props truyen tu thang cha, truyen tham so can truyen len thang cha
```

Comment:
Please don't use vietnamese to comment or code anymore. Also let comment at the top of function.

Reply:
Still vietnamese here